src/components/TokenExpired.js

```diff
@@ -1,25 +1,91 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+import { SessionTimeout } from 'patternfly-react'
 
 import ErrorAlert from './ErrorAlert'
 import { msg } from '_/intl'
 
+import { logout } from '_/actions'
+
+const MAX_INACTIVE_TIME = 60 * 20 // 20 minutes
+const TIME_TO_DISPLAY_MODAL = 30 // 30 seconds
```

Comment:
These settings should be at least in `ovirt-web-ui.config` and `src/config.js`  so they can be set on the server independently of a code change.  Ideally, a rest call can pull the ovirt-engine configured user session timeout, maybe even the one from webadmin.

Nit-pick: Use the expected time unit in the constant's name.  E.g. `MAX_INACTIVE_TIME_SEC`